### DISCUSSION TITLE
Fixed positioning swup container element

### DISCUSSION THREAD

**Discussion by qvilleret** [swup/swup#348] [Help] [answered]
Hi there, 

Thanks a lot for building and contributing on Swup js — I love it.

I've got a title on some pages - in the swup container - which I'd like to be fixed when scrolling. It looks like the css attributes does not work when coming from a swup link.

CSS :
![image](https://user-images.githubusercontent.com/7737305/111485650-16c6a400-8737-11eb-81f5-45da742e1643.png)

I've also tried to add css when the content is replaced (works as showing in the html). But won't work either.

![image](https://user-images.githubusercontent.com/7737305/111485855-470e4280-8737-11eb-87e5-49bb1b576c4f.png)

Any ideas on how to get this working ?

Thanks a lot for you help.

**Comment by daun** [comment, marked as answer]
Do you have a `transform` on the parent somewhere? `position: fixed` has tricky behavior inside transformed elements.

**Reply by qvilleret** [reply]
Exactly that..! the parent element had a transform animation, which was removing the fixed position. Thanks a lot @daun.
